lib/rules/no-textarea-mustache.js

```diff
@@ -23,7 +23,10 @@ module.exports = {
     return utils.defineTemplateBodyVisitor(context, {
       /** @param {VExpressionContainer} node */
       "VElement[name='textarea'] VExpressionContainer"(node) {
-        if (node.parent.type !== 'VElement') {
+        if (
+          node.parent.type !== 'VElement' ||
+          node.parent.rawName.startsWith('T')
```

Comment:
```suggestion
          node.parent.rawName[0] === 'T'
```

Reply:
I think using a named method should be easier than using notations to read.

What's your starting point about this?

Reply:
How about rewriting the selector?

```diff
-      "VElement[name='textarea'] VExpressionContainer"(node) {
+      "VElement[rawName ='textarea'] VExpressionContainer"(node) {`
```

Reply:
LGTM~

Reply:
Ah, even better!